docs/configuration.adoc

```diff
@@ -195,183 +195,219 @@ The following script levels are reserved by pre-defined scripts:
+----
+===== Cluster Configuration
+Clusters *cannot* use the MapDB key-value storage or the local filesystem for blob storage. They should configure the cluster.quorumMembers property in inetsoft.yaml.
+
+The quorum members contain the identifiers of the server and scheduler instances that are members of the cluster quorum. For any node in the cluster, at least (n / 2) + 1 quorum members must be visible to a node in order for that node to make any persistent changes. This prevents inconsistencies from being introduced during a split-brain condition due to network partitioning. The quorum member identifier for a server or scheduler instance is set by using the inetsoft.cluster.quorumMemberId system property when starting the instance.
+
+For statically provisioned clusters, all server and scheduler nodes should be included in the quorum.
+
+For elastic clusters, the "base" server and scheduler nodes should be included in the quorum. For example, if the minimum number of server nodes is configured to be 3 in a Kubernetes stateful set, the hostnames of the first three server pods and the hostnames for all the scheduler nodes should be included in the quorum. During normal operation these pods will always be running and provide a consistent cluster quorum.
+
+For example, a statically provisioned cluster with 3 server instances and 1 scheduler instance should be configured using:
+
+.inetsoft.yaml
+[source,yaml]
+cluster:
+  quorumMembers:
+    - server1
+    - server2
+    - server3
+    - scheduler
+
+The instances would be started with the following command line options, respectively:
+
+[source]
+-Dinetsoft.cluster.quorumMemberId=server1
+-Dinetsoft.cluster.quorumMemberId=server2
+-Dinetsoft.cluster.quorumMemberId=server3
+-Dinetsoft.cluster.quorumMemberId=scheduler
+
+A Kubernetes deployment a minimum number of server pods of 3, 1 scheduler pod and the name of the server stateful set server in the namespace inetsoft would use the configuration:
+
+.inetsoft.yaml
+[source,yaml]
+cluster:
+  quorumMembers:
+    - server-0.inetsoft.svc.cluster.local
+    - server-1.inetsoft.svc.cluster.local
+    - server-2.inetsoft.svc.cluster.local
+    - scheduler
+
+The server pods would have the following command line option:
+[source]
+-Dinetsoft.cluster.quorumMemberId=$(hostname)
+
+The scheduler pod would have the following command line option:
+[source]
+-Dinetsoft.cluster.quorumMemberId=scheduler
+
+==== Configure Asset Storage
+By default, inetsoft.yaml is configured using the internal database for the document node store and a file blob store with sensible file paths. If you want to use a different storage system, you'll need to include a custom `config/inetsoft.yaml` file in your external configuration.
+
+The inetsoft.yaml asset storage configuration has the following structure:
+
+.inetsoft.yaml
+[source,yaml]
+----
+#Default
+keyValue:
+  type: "mapdb" # <1>
+  mapdb: # <2>
+    directory: "/data_dir/kv" # <3>
+blob: # <4>
+  type: "local" # <5>
```

Comment:
This document is for Docker and Kubernetes. It shouldn't include an example of local. Instead, it should have an example of using a shared filesystem.